Source/Project64/User Interface/Notification Class.h

```diff
@@ -35,6 +35,8 @@ class CNotification :
 	//User Feedback
 	virtual void DisplayMessage(int DisplayTime, const wchar_t * Message) const;
 	virtual void DisplayMessage(int DisplayTime, LanguageStringID StringID) const;
+	virtual void DisplayMessageWide(int DisplayTime, const wchar_t * Message, ...) const;

```

Comment:
Yes, I saw that you removed it and I suppose you have a good reason for that. The only problem is that wide strings aren't displayed properly anymore. I tried this using an earlier version and didn't have this problem.

Is there a better way to solve this than just add a duplicate function?